src/styles/card.scss

```diff
@@ -0,0 +1,122 @@
+.card {
+  width: 198px;
+  background-color: white;
+  border: 1px solid $border-color;
+  border-radius: 5px;
+  font-family: $roboto-font;
+  color: $text-color;
+  font-weight: 400;
+  transition: all 300ms linear;
+
+  &:hover {
+    transform: scale(1.2);
+  }
+
+  &__container {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    padding: 16px;
+  }
+
+  &__name {
+    margin-top: 40px;
+  }
+
+  &__title {
+    font-size: $headline-size;
+    line-height: 18px;
+    font-weight: 500;
+    margin: 0;
+    color: $text-color;
+    transition: all 300ms linear;
+  }
+
+  &:hover &__title {
+    color: #34568b;
+    transition: all 300ms linear;
```

Comment:
You have a 'transition: all 300ms linear;' property set on the .card:hover &__title class. But you have already set the transition property in the .card &__title class. There's no need to repeat the transition property in the hover state. Remove it to keep your code DRY (Don't Repeat Yourself).